protocol-units/settlement/mcr/client/src/eth_client.rs

```diff
@@ -126,6 +128,7 @@ impl
 
 impl<P> Client<P> {
 	async fn build_with_provider<S>(
+		run_commitment_admin_mode: bool,
```

Comment:
It's easier to deal with like this. Otherwise, we don't have a nice mapping from the config or we have to update our macros to support parsing to enums. I will look into it later, but I would suggest it's not needed for this PR.